spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -367,6 +367,22 @@
     end
   end
 
+  describe '#parse_date' do
+    let(:response) { described_class.new(success_response, false, config) }
+
+    it 'has an invalid month' do
+      expect { response.send(:parse_date, year: 2022, month: 13, day: 1) }.not_to raise_error
+    end
+
+    it 'has an invalid leap day' do
+      expect { response.send(:parse_date, year: 2022, month: 2, day: 29) }.not_to raise_error
+    end
+
+    it 'has a day past the end of the month' do
+      expect { response.send(:parse_date, year: 2022, month: 4, day: 31) }.not_to raise_error
+    end
```

Comment:
Maybe we place an expectation on the `nil` response and that we log (since any exceptions thrown would fail the spec)

```suggestion
    it 'handles an invalid month' do
      expect(Rails.logger).to receive(:info)
      expect(response.send(:parse_date, year: 2022, month: 13, day: 1)).to eq(nil)
    end

    it 'handles an invalid leap day' do
      expect(Rails.logger).to receive(:info)
      expect(response.send(:parse_date, year: 2022, month: 2, day: 29)).to eq(nil)
    end

    it 'handles a day past the end of the month' do
      expect(Rails.logger).to receive(:info)
      expect(response.send(:parse_date, year: 2022, month: 4, day: 31) }).to eq(nil)
    end
```